subprojects/performance/templates.gradle

```diff
@@ -308,6 +301,29 @@ task bigCppMulti(type: CppMultiProjectGeneratorTask) {
     subProjectTemplates = ['cpp-source', 'cpp-project']
 }
 
+// Build-builder projects
+task buildBuilder(type: RemoteProject) {
+    remoteUri = 'https://github.com/adammurdoch/build-builder.git'
+    ref = 'ef909790b6d4311609af0cb53373b1c8a9180718'
+}
+
+task installBuildBuilder(type: GradleBuild) {
+    dir = buildBuilder.outputDirectory
+    tasks = ['install']
+    dependsOn buildBuilder
+}
+
+tasks.withType(BuildBuilderGenerator) {
+    buildBuilderInstall = new File(tasks.buildBuilder.outputDirectory, "build/install/build-builder")
+    dependsOn installBuildBuilder
+}
+
+task mediumSwiftMulti(type: BuildBuilderGenerator) {
```

Comment:
It would be good to add these task type in the task group `Project setup` so they can be discovered with `./gradlew :performance:tasks`. See `gradle/performanceTest.gradle` line 64:
```
tasks.withType(ProjectGeneratorTask) {
    group = 'Project setup'
    if(project.hasProperty("maxProjects")) {
        projects = maxProjects as Integer
    }
}
```

Reply:
Good catch.  I didn't think we had groups for these because it wasn't in `templates.gradle`.